plugins/rbac/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export { rbacPlugin, RbacPage, Administration } from './plugin';
 
-export { default as AdminPanelSettingsOutlinedIcon } from '@mui/icons-material/AdminPanelSettingsOutlined';
+export { default as RbacIcon } from '@mui/icons-material/VpnKeyOutlined';
```

Comment:
let's ensure that it's backword compatible and not remove any exports i.e keep old one add new one as well and maybe make a comment as deprecated for now WDYT?